website/src/components/Footer.js

```diff
@@ -1,8 +1,14 @@
 import cx from 'classnames';
+import GlobalsContext from 'contexts/GlobalsContext';
+import _get from 'lodash/get';
 import Link from 'next/link';
+import { useContext } from 'react';
 import styles from './Footer.module.scss';
 
 export default function Footer() {
+  const globals = useContext(GlobalsContext);
+  const menus = _get(globals, 'menus.nodes[0].menuItems.nodes', []);
```

Comment:
Do we need to introduce lodash for this? `menus?.nodes[0]?.menuItems?.nodes ?? []`

Reply:
Lodash is already present in the repo (presumably Next itself loads it) and I find the question mark operator doesn't work great for array indexes